www/README.md

```diff
@@ -1,44 +1,86 @@
-# Website
+# Skip Documentation Website
 
-The documentation website is built using [Docusaurus](https://docusaurus.io/).
+This website is built using [Docusaurus](https://docusaurus.io/) and hosts both our documentation and blog.
 
+## API Documentation
 
-# API Documentation
+The API documentation is generated using:
+- [TypeDoc](https://typedoc.org/index.html) for API documentation generation
+- [docusaurus-plugin-typedoc](https://typedoc-plugin-markdown.org/plugins/docusaurus) for Docusaurus integration
 
-The API docs are generated by [TypeDoc](https://typedoc.org/index.html) and integrated into docusaurus using [docusaurus-plugin-typedoc](https://typedoc-plugin-markdown.org/plugins/docusaurus).
-This setup needs the development/source docs site to be in the same repo as the sources, as there are relative paths in the docusaurus config file to the sources.
-A separate deployment step will be needed to generate the docs and push them to the repo that feeds the live site.
-This also means that we can iterate on the docs in the source repo without publishing them immediately, and then choose to publish them in sync with publishing packages.
+### Development Setup
 
-The workflow for working on API docs is:
-```
-- $ cd /path/to/repo/root
-- $ make docs                           # to initialize the api docs
-- $ make docs-run                       # to run the docs site locally
-- visit http://localhost:3000/docs/api/core
-- edit the doc comments in the sources
-- $ make docs                           # to regenerate the api docs
+The development/source docs site must be in the same repository as the source code due to relative paths in the Docusaurus configuration. This setup allows us to:
+- Iterate on documentation in the source repository
+- Choose when to publish documentation
+- Keep documentation in sync with package releases
+
+### Development Workflow
+
+#### Standard Workflow
+```bash
+# From repository root INSIDE your Docker instance
+make docs           # Initialize API documentation
 ```
-A faster but not robust way to regenerate the api docs is:
+
+```bash
+# From repository rood OUTSIDE your Docker instance
+make docs-run       # Run the docs site locally
+# Visit http://localhost:3000/docs/api/core or http://localhost/blog
+
+# After editing doc comments in the sources
+make docs           # Regenerate API documentation
 ```
-- cd /path/to/repo/root/www && npx docusaurus generate-typedoc
+
+### Publishing Documentation
+
+Before, commit your changes and send them for review
+```bash
+git add <your_files>
+git commit -m '<some message>'
+gh pr create # if you use the GitHub CLI
 ```
-To publish the docs to the live site:
+
+When reviewed and mereged, rebase and clean up your repository:
+```bash
+git pull --rebase
+git clean -Xdn # -n for a dry-run and double-check what's to be cleaned out
+git clean -Xdf # the actual cleaning
```

Comment:
Note that this is not normally needed, and it not specific to the docs. The issue you hit would have broken any of the builds in your checkout at that point.